src/main/java/com/training/api/Weight.java

```diff
@@ -0,0 +1,26 @@
+package com.training.api;
+
+import lombok.Getter;
+
+@Getter
+public class Weight {
+    private String imperial;
+    private String metric;
+
+    public String getImperial() {
```

Comment:
if you have used annotation  @getter then why their is use of Pojo(i.e getter ,setter)